package.json

```diff
@@ -1,26 +1,29 @@
 {
   "name": "@first-lego-league/ms-client",
-  "version": "1.0.3",
   "description": "A HTTP client working by the Module Standard",
+  "version": "1.0.3",
   "main": "index.js",
-  "repository": "git@github.com:FirstLegoLeague/ms-client.git",
+  "repository": "https://github.com/FirstLegoLeague/ms-client.git",
   "author": "Idan Stark <idanstark42@gmail.com>",
-  "license": "MIT",
+  "license": "GPL-3.0",
+  "files": [
+    "index.js",
+    "lib/**/*.js"
+  ],
   "engines": {
-    "node": ">=8.2.1"
+    "node": ">=8.0.0"
   },
   "scripts": {
-    "test": "cross-env nyc --all --reporter=text --check-coverage --lines 90 mocha --recursive",
+    "test": "cross-env nyc --all --reporter=text --check-coverage --lines 90 mocha --recursive --require mock-local-storage",

```

Comment:
remove cross-env (you don't use it here)

Reply:
Why not? I copied the scripts from `ms-logger` and it uses it there.